src/sentry/organizations/absolute_url.py

```diff
@@ -94,10 +94,11 @@ def organization_absolute_url(
 def api_absolute_url(
     *, slug: str, path: str, query: str | None = None, fragment: str | None = None
 ) -> str:
-    if path and path[0] != "/":
-        path = f"/{path}"
-    path = f"/api/0/organizations/{slug}{path}"
-    uri = absolute_uri(path)
+    if path.startswith("/"):
+        path = path.lstrip("/")
```

Comment:
nit: can skip this conditional since this lstrip won't affect a string that doesn't start with `/`

i.e. this should work fine
```python
full_path = f"/api/0/organizations/{slug}/{path.lstrip('/'}/"
```